template/xwiki/docker-entrypoint.sh

```diff
@@ -127,6 +127,10 @@ function configure() {
   safesed "replacepassword" \$DB_PASSWORD /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/hibernate.cfg.xml
   safesed "replacecontainer" \$DB_HOST /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/hibernate.cfg.xml
   safesed "replacedatabase" \$DB_DATABASE /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/hibernate.cfg.xml
+  
+ if [ "$DB_DATABASE" != "xwiki" ]; then
+    safesed "# xwiki.db=xwiki" "xwiki.db=$DB_DATABASE" /usr/local/tomcat/webapps/\$CONTEXT_PATH/WEB-INF/xwiki.cfg
+
```

Comment:
This seems wrong since it's missing an "end" to the "if".

Did you try? Can you confirm that it works?

Also you shouldn't use safesed to set a property in xwiki.cfg. If you check docker-entrypoint.sh you'll see other examples of setting properties in xwiki.cfg, for ex at https://github.com/xwiki-contrib/docker-xwiki/blob/a0191d869d4e9a0a779569721bd05518a0c650c0/template/xwiki/docker-entrypoint.sh#L120

Thanks